### PR TITLE
add mdsnippets

### PR DESCRIPTION
**Pull request by SimonCropp**
do u want a GH action as well? https://github.com/SimonCropp/MarkdownSnippets/blob/master/docs/github-action.md this way when people edit files online on GH, doc will auto update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
-    pic1.InitializeDragAndDrop()
-        .Copy()                                                   // Copy(), Move() or Link() to define allowed effects
-        .Immediately()                                            // or OnMouseMove() for deferred start on mouse move
-        .WithData(pic1.Image)                                     // pass any object you like
-        .WithPreview(img => Watermark(img)).RelativeToCursor()    // define your preview and how it should behave
-        .To(pic2, (target, data) => target.Image = data);         // use your data after it was dropped (with type safety)
-
+    var pic = sender as PictureBox;
+    pic.InitializeDragAndDrop()
+        // Copy(), Move() or Link() to define allowed effects
```
Comment (SimonCropp) [inline README.md:16]: comments above the line render much better on mobile devices

File: README.md [hunk 2]
```diff
@@ -1,41 +1,52 @@
...
 private void pic1_MouseDown(object sender, MouseEventArgs e)
 {
-    pic1.InitializeDragAndDrop()
-        .Copy()                                                   // Copy(), Move() or Link() to define allowed effects
-        .Immediately()                                            // or OnMouseMove() for deferred start on mouse move
-        .WithData(pic1.Image)                                     // pass any object you like
-        .WithPreview(img => Watermark(img)).RelativeToCursor()    // define your preview and how it should behave
-        .To(pic2, (target, data) => target.Image = data);         // use your data after it was dropped (with type safety)
-
+    var pic = (PictureBox) sender;
```
Comment (SimonCropp) [inline README.md:14]: should only us `as` when you are unsure of the type

File: README.md [hunk 3]
```diff
@@ -1,41 +1,52 @@
 # Fluent Drag&Drop
 
+[![NuGet Status](https://img.shields.io/nuget/v/FluentDragDrop.svg)](https://www.nuget.org/packages/FluentDragDrop/)
```
Comment (SimonCropp) [inline README.md:3]: let me know if u dont want a nuget shield

Comment (Lakritzator) [reply README.md:3]: NuGet shields are a must 👍

**Review by awaescher — APPROVED** [review]

**Pull request merged**